app/helpers/templateGenerator.js

```diff
@@ -1,21 +1,43 @@
 const fs = require('fs');
 
-const getInputFields = (params) => {
-  const paramsInputString = Object.entries(params).map(([key, value]) => {
-    const modifiedValue = typeof value === 'string'
-      ? value.replace(/&lt/g, '<').replace(/&gt;/g, '>')
-      : value;
-    return `
-      <div class="nhsuk-u-inline-block nhsuk-u-padding-4 nhsuk-u-padding-bottom-0">
-        <label for="param" class="nhsuk-grid-column-one-quarter">${key}:</label>
-        <input type="text" name="${key}" id="${key}" value="${modifiedValue}" class="nhsuk-grid-column-three-quarters nhsuk-u-font-size-16"/>
-      </div>
-    `;
-  }).join('');
-  return paramsInputString;
+const stringConstructor = 'test'.constructor;
+const arrayConstructor = [].constructor;
+const objectConstructor = ({}).constructor;
+
+const objectSize = (object) => {
+  let size = 0;
+  Object.keys(object).forEach(() => {
+    size += 1;
+  });
+  return size;
+};
+
+const whatIsIt = (object) => {
+  if (object === null) {
+    return 'null';
+  }
+  if (object === undefined) {
+    return 'undefined';
+  }
+  if (object.constructor === stringConstructor) {
+    return 'String';
+  }
+  if (object.constructor === arrayConstructor) {
+    return 'Array';
+  }
+  if (object.constructor === objectConstructor) {
+    return 'Object';
+  }
+  return 'unknown';
 };
```

Comment:
I don't quite understand what this function or its purpose and the 3 consts at the start are doing here